pkg/interfaces/contracts/vault/IVaultExtension.sol

```diff
@@ -67,6 +67,12 @@ interface IVaultExtension {
      */
     function getReservesOf(IERC20 token) external view returns (uint256);
 
+    /**
+     * @notice Returns `true` if `addLiquidity` was called for the given pool in this transaction.
+     * @param pool Address of the pool to check.
```

Comment:
```suggestion
     * @notice This flag is used to detect and tax "round trip" transactions (adding/removing liquidity in the same pool)
     * @param pool Address of the pool to check
     * @return liquidityAdded True if liquidity has been added to this pool in the current transaction
```

Reply:
Could also add a dev section here with more detail on why this is desirable / improves security, and documenting limitations.